catalog-info.yaml

```
@@ -14,7 +14,7 @@ metadata:
   annotations:
     # (Optional) We use the below annotation to indicate whether or not this
     # repository should be tagged for openedx releases and which branch is tagged.
-    openedx.org/release: null
+    openedx.org/release: "master"
```

```suggestion
    openedx.org/release: null
```

I think we actually don't need to tag this repository.  We used to for some reason but I don't think that makes sense.